pkgs/development/python-modules/prometheus-pve-exporter/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  prometheus-client,
+  proxmoxer,
+  pyyaml,
+  requests,
+  werkzeug
+}:
+
+buildPythonPackage rec {
```

Comment:
Use `buildPythonApplication` instead and move the file into `pkgs/servers/monitoring/prometheus.nix` instead. Then move the reference in `python-packages.nix` into `all-packages.nix`.